Python/compile.c

```diff
@@ -8650,29 +8650,22 @@ optimize_basic_block(struct compiler *c, basicblock *bb, PyObject *consts)
             }
 
                 /* Try to fold tuples of constants.
-                   Skip over BUILD_SEQN 1 UNPACK_SEQN 1.
-                   Replace BUILD_SEQN 2 UNPACK_SEQN 2 with ROT2.
-                   Replace BUILD_SEQN 3 UNPACK_SEQN 3 with ROT3 ROT2. */
+                   Skip over BUILD_TUPLE(1) UNPACK_SEQUENCE(1).
+                   Replace BUILD_TUPLE(2) UNPACK_SEQUENCE(2) with SWAP(2).
+                   Replace BUILD_TUPLE(3) UNPACK_SEQUENCE(3) with SWAP(3). */
             case BUILD_TUPLE:
                 if (nextop == UNPACK_SEQUENCE && oparg == bb->b_instr[i+1].i_oparg) {
                     switch(oparg) {
                         case 1:
                             inst->i_opcode = NOP;
                             bb->b_instr[i+1].i_opcode = NOP;
-                            break;
+                            continue;
```

Comment:
Why change this? Aren't they equivalent.
Having a `continue` here distracts from the flow of the code, as it is not the normal way to end a `case` and increases cognitive load (at least it did for me).

Reply:
They’re not quite equivalent: note that I also removed the `break` on old line 8675.

Previously, values other than 1, 2, or 3 wouldn’t get a chance to hit `fold_tuple_on_constants`. Now they do.

Reply:
:+1: LGTM then.